packages/utils/feed/map.ts

```diff
@@ -6,14 +6,22 @@ import musicPostSvg from "@/assets/icons/social-feed-map/music-post.svg";
 import normalPostSvg from "@/assets/icons/social-feed-map/normal-post.svg";
 import picturePostSvg from "@/assets/icons/social-feed-map/picture-post.svg";
 import videoPostSvg from "@/assets/icons/social-feed-map/video-post.svg";
-import { CustomLatLngExpression, PostCategory } from "@/utils/types/feed";
+import {
+  CustomLatLngExpression,
+  LatLngBoundExpression,
+  PostCategory,
+} from "@/utils/types/feed";
 
 //TODO: Get <svg/> as string from svg files (These svg strings are used in FeedMapList.web.tsx)
 
 export const MAP_LAYER_URL = `https://{s}.tile.jawg.io/jawg-dark/{z}/{x}/{y}{r}.png?access-token=${process.env.EXPO_PUBLIC_LEAFLET_MAP_TOKEN}`;
 
 // Paris baguette
-export const DEFAULT_MAP_POSITION: CustomLatLngExpression = [48.8566, 2.3522];
+export const DEFAULT_MAP_POSITION: CustomLatLngExpression = [20, 0];
```

Comment:
Please remove remove this change to keep the PR atomic, it's out of scope.
You made this fix in this PR: https://github.com/TERITORI/teritori-dapp/pull/1398/files#diff-a44b6f872b57f327a6fa7ab9ba11100753e475ce41790a0b6b20867e19984addR16

Also, the default position is wrong. But it doesn't concern this PR
![image](https://github.com/user-attachments/assets/e47eea81-30df-46f4-a9f5-c75948d50223)